### ISSUE TITLE
Grails 6 war deployment on  Tomcat 8 gives Spring security login page on accessing the app , where as if i run locally using grails r-a it works fine

### ISSUE THREAD

**Issue by bishtmohan7** [grails/grails-core#13170]
### Expected Behavior

![Screenshot from 2023-10-17 13-57-32](https://github.com/grails/grails-core/assets/12708027/5fcca403-82ec-46cd-b0e1-712ca451dcc9)

### Version

6.0.0

**Comment by puneetbehl** [comment]
Can you please share a sample Grails application?

**Comment by jeffscottbrown** [comment]
> Grails 6 war deployment on Tomcat 8 gives Spring security login page on accessing the app , where as if i run locally using grails r-a it works fine

@bishtmohan7 Are you running the Grails 6 app locally with "grails run-app"?

**Comment by bishtmohan7** [comment]
@jeffbrown yes , it works fine with grails run-app , give issues as soon as we create a war and deploy in tomcat

**Comment by bishtmohan7** [comment]
@puneetbehl i got this issue when i was migrating grails 4 app to grails 6  and i was using spring security  ,I tried to create sample grails 6 application , there it gives issues while deploying war on tomcat 8 ,seems like not supporting external tomcat while it works fine with embedded tomcat (run-app )

**Comment by puneetbehl** [comment]
Could you please share the link to the GitHub repository for your sample application? Also, it would be helpful if you could also share the exact version of Tomcat 8.

**Comment by rkrishnas81** [comment]
I am trying Grails 6 jdk17  on tomcat 
Please share hello world app .
It's so helpful
